hervebutera/my_brand-backend
# Add APIs for creating a user, logging in a user and updating the user's info

What does this PR do?
This PR adds API endpoints that allow signing up, logging in, getting user info, and updating user info

Description of Task to be completed?
Adds API endpoint for signing up a new user
Adds API endpoint for retrieving logged in user's info
Adds API endpoint for updating logged in user's info in the MongoDB

How should this be manually tested?
Clone the repository and install all the necessary packages as shown by the package.json file,
then run npm start.
Also use postman to check if they pass.

Any background context you want to provide?
Validation before adding and updating user's info to mongoDB was included.
Check for user token to log in and update user's info in the MongoDB

Screenshots (if appropriate
